ci/docker/runtime_functions.sh

```diff
@@ -40,19 +40,24 @@ build_jetson() {
     pushd .
     mv make/crosscompile.jetson.mk make/config.mk
     make -j$(nproc)
-
     export MXNET_LIBRARY_PATH=`pwd`/libmxnet.so
     cd /work/mxnet/python
-    python setup.py bdist_wheel --universal
+    build_wheel
+    popd
+}
 
-    # Fix pathing issues in the wheel.  We need to move libmxnet.so from the data folder to the
-    # mxnet folder, then repackage the wheel.
+build_wheel() {
+    set -ex
+    pushd .
+    python setup.py bdist_wheel --universal
     WHEEL=`readlink -f dist/*.whl`
     TMPDIR=`mktemp -d`
     unzip -d $TMPDIR $WHEEL
     rm $WHEEL
     cd $TMPDIR
     mv *.data/data/mxnet/libmxnet.so mxnet
+    cd $TMPDIR
+    #zip -r $WHEEL $TMPDIR
```

Comment:
Maybe choose one style? I think ```zip -r $WHEEL $TMPDIR``` is good

Reply:
I think there were problems, this works.